### PR TITLE
Fix Heise

### PR DESCRIPTION
**Pull request by addie9800**
Heise had a minor layout update one week ago. This PR fixes it by changing a minor detail. As it turns out Heise has a gadget blog called techstage and I added support for that as well. Example link: https://www.techstage.de/test/mini-pc-fuer-190-euro-mit-seltenem-extra-ouvis-gk3-plus-mit-vga-anschluss-im-test/lllvhkg

### CODE REVIEW

**Review by MaxDall — CHANGES_REQUESTED** [review]
Since the layout change a new version would be necessary to parse previously crawled articles.

**Comment by addie9800** [comment]
> Since the layout change a new version would be necessary to parse previously crawled articles.

Actually, I did add a new version at first, but then I did decide against it, since the selectors now are just a generalized version of the previous ones and should be completely backwards compatible. The subheadline and summary selectors were just extended and the paragraph selector was also just extended and the existing part generalized.

File: src/fundus/publishers/de/heise.py [hunk 1]
```diff
@@ -30,7 +30,18 @@ class V1(BaseParser):
...
+            "//div[@class='ringCommonDetail ringBlockType-paragraph ']/p[not(@class"
+            " or ((string-length(text()) < 3) and (contains(text(), '(') or contains(span, '(')))"
+            " or contains(text(), '=== Anzeige / Sponsorenhinweis')"
+            " or contains(text(), 'Tipp: Wir sind bei WhatsApp!')"
+            " or contains(a, 'heise+ abonnieren')"
+            " or contains(text(), '► '))"
+            " or @class='antwort rte__abs--antwort'"
+            " or @class='frage rte__abs--frage'"
+            " or @class='json-ld-paid-content-marker'] "
+            " | //div[@class='ringCommonDetail ringBlockType-paragraph ']//ul["
```
Comment (MaxDall) [inline src/fundus/publishers/de/heise.py:44]: Is this all `techstage` related?

Comment (addie9800) [reply src/fundus/publishers/de/heise.py:44]: Yes, they use a slightly different formatting.

Comment (addie9800) [reply src/fundus/publishers/de/heise.py]: But I did realise that the selectors can be simplified, as techstage does not seem to contain as much bloat as heise

**Review by MaxDall — APPROVED** [review]

**Pull request merged**